src/main/js/components/Customer.js

```diff
@@ -18,13 +24,80 @@ import ReactDOM from "react-dom";
  *
  */
 export class Customer extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {pageSize: 10, selectedView: 'Customer'}
+    }
+
+    componentDidMount(){
+        this.props.loadResourceFromServer('menuItems', this.state.pageSize);
+        this.props.loadResourceFromServer('diningSessions', this.state.pageSize);
+    }
+
     render() {
         return (
-            <CustomerLanding /> // TODO: Clicking this should move the customer to the Menu page?
-        )
+            <TableNumberSelect
+                handleTableNumberSelect={this.handleTableNumberSelect}
+                diningSessions={this.props.diningSessions}
+                diningSessionAttributes={this.props.diningSessionAttributes}
+                history={this.props.history}
+                onUpdate={this.props.onUpdate}/>
+        );
     }
 }
 
+class TableNumberSelect extends React.Component{
+
+    constructor(props){
+        super(props);
+        this.handleTableNumberSelect = this.handleTableNumberSelect.bind(this);
+    }
+
+
+    handleTableNumberSelect(e) {
+        e.preventDefault();
+        const updatedDiningSession = {};
+
+        let selectedTableNumber = document.getElementById('table-select-dropdown').value
+
+        updatedDiningSession['tableNumber'] = selectedTableNumber;
+        updatedDiningSession['diningSessionStatus'] = 'ACTIVE';
+        updatedDiningSession['serviceRequestStatus'] = 'INACTIVE';
+        updatedDiningSession['billRequestStatus'] = 'INACTIVE';
+
+        let oldDiningSession = this.props.diningSessions.find(function(session) {
+            return session.entity.tableNumber === parseInt(selectedTableNumber, 10);
+        });
+
+        this.props.onUpdate(oldDiningSession, updatedDiningSession, 'diningSessions');
+        this.props.history.push('/CustomerLanding');
```

Comment:
When we push the customerLanding page onto the history, how are the props propagated?

My specific issue is that looking long-term when the customer creates an order they will need to know their diningsession, but there can be multiple active.

As such I think the customerLanding should have an additional property **_tableNumber_**. So that way the customer when creating an order can access its dining session via `this.props.tableNumber` and then parse the `diningsessions` for the corresponding **_tableNumber_**. 

However with the current implementation I don't see how we can know which dining session corresponds to the active tablet on which the session is ran.

Reply:
Sounds good with me